apps/sms/views/conversation/js/information.js

```diff
@@ -227,7 +227,8 @@ function getOperatorByServiceId(index) {
 
 var VIEWS = {
   group: {
-    name: 'participants',
+    name: 'group-view',
```

Comment:
Okay will take care about Participants ---> Group migration in integration tests :)